test/functional/sbom/test_cycloneDX.py

```diff
@@ -0,0 +1,139 @@
+import textwrap
+
+import pytest
+
+from conan.test.assets.genconanfile import GenConanfile
+from conan.test.utils.tools import TestClient
+from conans.util.files import save
+import os
+
+# Using the sbom tool with "conan create"
+sbom_hook_post_package = """
+import json
+import os
+from conan.errors import ConanException
+from conan.api.output import ConanOutput
+from conan.tools.sbom.cycloneDX import cyclonedx_1_4
+
+def post_package(conanfile):
+    try:
+        sbom_cyclonedx_1_4 = cyclonedx_1_4(conanfile.subgraph)
+        metadata_folder = conanfile.package_metadata_folder
+        file_name = "cyclonedx_1_4.json"
+        with open(os.path.join(metadata_folder, file_name), 'w') as f:
+            json.dump(sbom_cyclonedx_1_4, f, indent=4)
+        ConanOutput().success(f"CYCLONEDX CREATED - {conanfile.package_metadata_folder}")
+    except Exception as e:
+        ConanException("error generating CYCLONEDX file")
+"""
+
+@pytest.fixture()
+def hook_setup_post_package():
+    tc = TestClient()
+    hook_path = os.path.join(tc.paths.hooks_path, "hook_sbom.py")
+    save(hook_path, sbom_hook_post_package)
+    return tc
+
+
+def test_sbom_generation_create(hook_setup_post_package):
+    tc = hook_setup_post_package
+    tc.run("new cmake_lib -d name=dep -d version=1.0")
```

Comment:
These tests are slow, better try to pick the ``transitive_libraries`` common fixture, so this doesn't keep increasing the test times